### DISCUSSION TITLE
Page not rendering

### DISCUSSION THREAD

**Discussion by pablotorresbakata** [thuliteio/doks#770] [Q&A]
When I do npm run start and starts the server the html renders correctly but when I build it with npm run build and open the index.html file with chrome the page dont render correctly. 

When I check the chrome console it says that the fonts cannot load becuase the CORS policy block them and other resources such as js files and images can not be loaded

**Comment by h-enk** [comment]
Do you have a repo you can share?

**Reply by pablotorresbakata** [reply]
This is the repo


https://github.com/pablotorresbakata/HugoTest

**Reply by h-enk** [reply]
Sorry for getting back this late (I've been a bit busy).

Cannot reproduce your error — see my [Netlify deploy](https://roaring-sawine-8fa918.netlify.app/) of your repo — have you fixed it yourself in the mean time?

Note, I got an error on `npm i` though — see https://github.com/pablotorresbakata/HugoTest/blob/master/package.json#L67-L71

I had to remove the blank line and make it not 0.99.1:

```json
..
  },
  "otherDependencies": {
    "hugo": "0.99.0"
  }
}
```

P.S. I will remove your repo + my Netlify deploy when you're good.
